changelog/issue-7443.md

```diff
@@ -0,0 +1,8 @@
+audience: worker-deployers
+level: minor
```

Comment:
Would this technically be a major change here since if someone has their config set to something less than 1200 before this change, after the change, they'll get a schema validation error, right?

Reply:
good question! I'm not sure, this is simply a validation from my perspective, so shouldn't break anything. 
Change my mind :)